extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -31,6 +31,14 @@ import {user} from '../../../../../src/log';
  */
 export let ArticleComponentDef;
 
+/**
+ * @struct @typedef {{
+ *   heading: !Element,
+ *   meta: !Element,
+ * }}
```

Comment:
Nit: Put this on one line. CC doesn't mind but GH and VSCode barf on multiline type definitions.